<p>
  Services with interface element 'REPEAT' need to be configured correctly.
</p>

<pre>
  Services with interface element 'REPEAT' step need to be configured correctly:
  	- The "Count" property needs to be configured.
  	- It is bad practice to leave the "Timeout" property empty 
</pre>

<h2>Noncompliant Code Example</h2>
<pre>
Any non-compliant variant
</pre>

<h2>Compliant Solution</h2>
<pre>
	- Configure the "Count" property.
  	- Configure the "Timeout" property.
</pre>