<p>
  The should be no invocations of pub.flow:clearPipeline in non-toplevel services
</p>  
<p>  
  The should be no invocations of pub.flow:clearPipeline because clearing the pipeline globally can have unwanted side-effects.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
  Invoke pub.flow:clearPipeline
</pre>

<h2>Compliant Solution</h2>
<pre>
  Remove invocation of pub.flow:clearPipeline and replace it with a MAP step to drop unwanted pipeline contents.
</pre>